core/templates/dev/head/css/oppia.css

```diff
@@ -348,6 +348,15 @@ p {
   word-wrap: break-word;
 }
 
+/* The properties should be in sync with the p tag so that we have consistent
+   spacing between the list, above the first bullet (point) and at the end of last
+   bullet (point). Line height has a value 1.846 and not 1.2 because all p tags
+   have value 1.846 and it comes from third party due to narrower css there.*/
+ul, ol {
```

Comment:
This will effect ul and ol tags everywhere on the site instead of just the rte. Is that expected?

Reply:
Yes, this is expected acc to me. I have mentioned the reason for the same here: https://github.com/oppia/oppia/pull/6554#issue-267352667. Let me know if you still have any questions :). Thanks!

Reply:
Ok, sounds good.